Formula/llvm.rb

```diff
@@ -188,17 +226,36 @@ def install
     args = %w[
       -DLLVM_OPTIMIZED_TABLEGEN=On
       -DLLVM_BUILD_LLVM_DYLIB=On
+      -DLLVM_TARGETS_TO_BUILD=X86
```

Comment:
Given the bottle is `cellar :any` is there actual user-facing harm in including all the targets & saving people who do want to use slightly more obscure ones the lengthy manual rebuild, given that from-source rebuild is kind of a huge deal?